pom.xml

```diff
@@ -33,7 +33,7 @@
     </parent>
     <groupId>org.jenkins-ci.plugins.workflow</groupId>
     <artifactId>workflow-api</artifactId>
-    <version>2.2-SNAPSHOT</version>
+    <version>2.2.blockapis-SNAPSHOT</version>
```

Comment:
FYI, the right way to do this kind of versioning would be `2.2-blockapis-SNAPSHOT` - `2.2.blockapis-SNAPSHOT` will actually be treated as a higher version than `2.2-SNAPSHOT`.